openapi.yaml

```diff
@@ -127,15 +127,15 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Incident'
+              $ref: '#/components/schemas/IncidentPost'
```

Comment:
why you added post here, if method already post?

Reply:
Because this schema relates to the POST requests. The `IncidentPost` and `Incident` are different schemas. The key differences are fields `Updates` (which is missing for POST) and `Description` that's optional for POST and is not present for schema `Incident`.

Reply:
I should somehow match data schema for requests. That's why POST and other postfix are related to given methods.